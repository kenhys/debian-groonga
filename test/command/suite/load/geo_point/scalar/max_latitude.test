table_create LandMarks TABLE_NO_KEY
column_create LandMarks location COLUMN_SCALAR WGS84GeoPoint

load --table LandMarks
[
{"location": "324000000x502419287"}
]

select LandMarks
