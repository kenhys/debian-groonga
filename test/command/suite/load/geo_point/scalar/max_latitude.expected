table_create LandMarks TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create LandMarks location COLUMN_SCALAR WGS84GeoPoint
[[0,0.0,0.0],true]
load --table LandMarks
[
{"location": "324000000x502419287"}
]
[[0,0.0,0.0],1]
select LandMarks
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "location",
          "WGS84GeoPoint"
        ]
      ],
      [
        1,
        "324000000x502419287"
      ]
    ]
  ]
]
